.github/workflows/build.yml

```diff
@@ -0,0 +1,70 @@
+name: Build
+
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened, ready_for_review, edited]
```

Comment:
If it's to build the code, let's not listen to all these. `opened` and `synchronize` are enough.